zebra-chain/src/transaction.rs

```diff
@@ -99,6 +99,21 @@ pub enum Transaction {
         /// The shielded data for this transaction, if any.
         shielded_data: Option<ShieldedData>,
     },
+    /// An Orchard transaction (`version = 5`).
+    V5 {
+        /// The transparent inputs to the transaction.
+        tx_in: Vec<transparent::Input>,
+        /// The transparent outputs from the transaction.
+        tx_out: Vec<transparent::Output>,
```

Comment:
Please match the names used by other `Transaction` variants:
```suggestion
        inputs: Vec<transparent::Input>,
        /// The transparent outputs from the transaction.
        outputs: Vec<transparent::Output>,
```

Reply:
I am not so sure about this. According to `7.1 Transaction Encoding and Consensus` from the orchard pdf the V4 and the V5 haves their inputs as `tx_in` and outputs as `tx_out`. Please clarify.

Reply:
We don't need to use exactly the same names as the spec, we just need to have identical parsing behaviour. Parsing does not depend on field names.

Occasionally we choose different names, so the code is easier to read. So we should try to match existing Zebra code first, then match the spec.